doc/rpc.rst

```diff
@@ -31,15 +31,22 @@ will work with all servers ``>= X``. However, it will not work with any servers
 ``< X``.
 
 The library contains a client implementation parameterized over a concurrency
```

Comment:
```suggestion
The library contains a client implementation parameterised over a concurrency
```

"s" in British spelling, unless this is a specific CS term that must be spelled that way.